source/openapi-admin-v3.yaml

```diff
@@ -1397,30 +1397,24 @@ paths:
       - "$ref": "#/components/parameters/AppId"
       - "$ref": "#/components/parameters/ServiceId"
       - "$ref": "#/components/parameters/IncomingWebhookId"
-  "/groups/{groupId}/apps/{appId}/sync/data":
```

Comment:
Is this endpoint still relevant for PBS? If so, should we keep it around and make it clear that FS uses the service config?